src/NuGet.Clients/NuGet.PackageManagement.PowerShellCmdlets/NuGetPowerShellBaseCommand.cs

```diff
@@ -68,14 +68,15 @@ public abstract class NuGetPowerShellBaseCommand : PSCmdlet, IPSNuGetProjectCont
 
         protected NuGetPowerShellBaseCommand()
         {
-            _sourceRepositoryProvider = ServiceLocator.GetComponentModelService<ISourceRepositoryProvider>();
-            ConfigSettings = ServiceLocator.GetComponentModelService<Configuration.ISettings>();
-            VsSolutionManager = ServiceLocator.GetComponentModelService<IVsSolutionManager>();
+            var componentModel = NuGetUIThreadHelper.JoinableTaskFactory.Run(ServiceLocator.GetComponentModelAsync);
```

Comment:
out of curiosity, why is `ServiceLocator.GetComponentModelAsync` run under `JTF.Run` context here but under `Task.Run` context in the `src/NuGet.Clients/NuGet.Console/Xamls/ConsoleContainer.xaml.cs` line no 52 in the same PR?
https://github.com/NuGet/NuGet.Client/pull/4236/files#diff-b12d9891ad5a019186d037b31f27ede01cebc658a1243629ade2bb5fc60b6ee3R52